base/test.jl

```diff
@@ -138,7 +138,7 @@ type Broken <: Result
     orig_expr
 end
 function Base.show(io::IO, t::Broken)
-    print_with_color(:yellow, io, "Test Broken\n")
+    print_with_color(:yellow, io, "Test Broken\n"; bold = true)
```

Comment:
answering my own question: not yet https://github.com/JuliaLang/julia/pull/18453